etc/profile-a-l/evince.profile

```diff
@@ -7,6 +7,7 @@ include evince.local
 include globals.local
 
 noblacklist ${HOME}/.config/evince
+noblacklist ${HOME}/.local/share/gvfs-metadata
```

Comment:
We don't seem to blacklist that path anywhere AFAICT, so this doesn't make much sense. Are you by any chance needing this somehow because of options in a evince.local file?

Reply:
hmm, that line I have added is the only line in my .local for evince. It doesn't work without it.
I will do more digging next week, maybe I have changed some global settings I can't remember.